### PR TITLE
JS Linting: use Prettier instead of ESlint warnings

### PR DESCRIPTION
**Pull request by caleybrock**
This fixes 2 issues
1.  we had incompatible dependency warnings after: https://github.com/code-for-canada/project-thundercat/pull/22
```
frontend_1  | warning " > eslint-config-airbnb-base@13.1.0" has unmet peer dependency "eslint@^4.19.1 || ^5.3.0".
frontend_1  | warning " > eslint-config-airbnb-base@13.1.0" has unmet peer dependency "eslint-plugin-import@^2.14.0".
```
2. linting caused a lot of errors and was pretty annoying. this change auto-fixes your lint warnings every time you save a file, but still gives you red underlines if you have an error.
ex - fixes style on "save" of file in VSCode: 
prettier
ex - still provides errors
screen shot 2019-01-29 at 12 33 44 pm

I followed this tutorial to get this working: https://blog.echobind.com/integrating-prettier-eslint-airbnb-style-guide-in-vscode-47f07b5d7d6a

All devs will need to follow additional steps in docs to get their VSCode setup properly.

### CODE REVIEW

**Comment by caleybrock** [comment]
This has a lot of lines of code changes, but there are no actual code changes.
Tests pass.
screen shot 2019-01-29 at 8 23 26 am

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,33 +1,28 @@
...
-              <Link to="/status">Status</Link>
-            </div>
-            <hr />
-            <Route exact path="/" component={Home} />
-            <Route path="/experiment" component={Experiment} />
-            <Route path="/status" component={Status} />
+            <Link to="/">Home</Link>
+            <Link to="/experiment">Prototype</Link>
+            <Link to="/status">Status</Link>
           </div>
```
Comment (fnormand01) [inline frontend/src/App.js]: Is that normal that the page names don't have any space between each others?

image

Comment (caleybrock) [reply frontend/src/App.js]: It's not great but we're going to be deleting this navigation soon anyways so I'm not too worried about it.

**Comment by caleybrock** [comment]
Got some feedback offline that this isn't working as expected, so I'll ping you both once I'm more confident, for another review.

**Comment by fnormand01** [comment]
Works fine on my side. Tested the app and everything is working well. Also ran the tests and everything passed.

**Pull request merged**